### PR TITLE
Fix for https://github.com/Yoast/yoast-acf-analysis/issues/26

### LINKED ISSUE #26: Select2 problem with ACF5 and Taxonomy multi-select field
Whenever a Taxonomy field is created, with multi-select as appearance, the following error is thrown:

`The select2('data') method was called on an element that is not using Select2.`

This is breaking all functionality, and no more data is parsed by the code.

Specifically for branch: https://github.com/Yoast/yoast-acf-analysis/tree/features/acf-5

/cc @kraftner

### PR DESCRIPTION
**Pull request by richardsweeney**
This fixes #26

### CODE REVIEW

File: js/yoast-acf-analysis.js [hunk 1]
```diff
@@ -807,4 +808,5 @@ var isTinyMCEAvailable = function(editorID) {
...
...bi8qKlxuICogQWRhcHRlZCBmcm9tIHdwLXNlby1zaG9ydGNvZGUtcGx1Z2luLTMwNS5qczoxMTUtMTI2XG4gKlxuICogQHJldHVybnMge3N0cmluZ31cbiAqL1xudmFyIGdldENvbnRlbnRUaW55TUNFID0gZnVuY3Rpb24oZmllbGQpIHtcbiAgICB2YXIgdGV4dGFyZWEgPSBmaWVsZC4kZWwuZmluZCgndGV4dGFyZWEnKVswXTtcblxuICAgIHZhciBlZGl0b3JJRCA9IHRleHRhcmVhLmlkO1xuXG4gICAgdmFyIHZhbCA9IHRleHRhcmVhLnZhbHVlO1xuXG4gICAgaWYgKCBpc1RpbnlNQ0VBdmFpbGFibGUoZWRpdG9ySUQpICkge1xuICAgICAgICB2YWwgPSB0aW55TUNFLmdldCggZWRpdG9ySUQgKSAmJiB0aW55TUNFLmdldCggZWRpdG9ySUQgKS5nZXRDb250ZW50KCkgfHwgJyc7XG4gICAgfVxuXG4gICAgcmV0dXJuIHZhbDtcbn07XG5cbi8qKlxuICogQWRhcHRlZCBmcm9tIHdwLXNlby1wb3N0LXNjcmFwZXItcGx1Z2luLTMxMC5qczoxOTYtMjEwXG4gKlxuICpcbiAqIEBwYXJhbSBlZGl0b3JJRFxuICogQHJldHVybnMge2Jvb2xlYW59XG4gKi9cbnZhciBpc1RpbnlNQ0VBdmFpbGFibGUgPSBmdW5jdGlvbihlZGl0b3JJRCkge1xuICAgIGlmICggdHlwZW9mIHRpbnlNQ0UgPT09ICd1bmRlZmluZWQnIHx8XG4gICAgICAgIHR5cGVvZiB0aW55TUNFLmVkaXRvcnMgPT09ICd1bmRlZmluZWQnIHx8XG4gICAgICAgIHRpbnlNQ0UuZWRpdG9ycy5sZW5ndGggPT09IDAgfHxcbiAgICAgICAgdGlueU1DRS5nZXQoIGVkaXRvcklEICkgPT09IG51bGwgfHxcbiAgICAgICAgdGlueU1DRS5nZXQoIGVkaXRvcklEICkuaXNIaWRkZW4oKSApIHtcbiAgICAgICAgcmV0dXJuIGZhbHNlO1xuICAgIH1cblxuICAgIHJldHVybiB0cnVlO1xufTtcblxubW9kdWxlLmV4cG9ydHMgPSBTY3JhcGVyOyJdfQ==
```
Comment (kraftner) [inline js/yoast-acf-analysis.js]: The source map shouldn't be included. Can you rebuild the file?

**Comment by richardsweeney** [comment]
@kraftner this is fixed now

**Comment by kraftner** [comment]
Thanks. This seems to fix the issue - but also breaks it for ACF<5.6 where everything was working fine. So it seems as if this error only appears in ACF>5.6 due to the switch from select2 v3 to v4. I think we should move this back to #26 for now to first discuss if we just say we only support ACF>5.6 or make a fix that works with both versions.

**Comment by kraftner** [comment]
I am sorry but I'm closing this in favour of #45 which works with all versions of ACF5. Thanks for your work anyway as it pointed me in the right direction!

**Pull request closed without merging**